src/components/Table/GridTable.tsx

```diff
@@ -1474,24 +1474,25 @@ function filterRows<R extends Kinded>(
   filter: string | undefined,
 ): ParentChildrenTuple<R>[] {
   // Make a function to do recursion
-  function filterFn(acc: ParentChildrenTuple<R>[], row: GridDataRow<R>): ParentChildrenTuple<R>[] {
+  function filterFn(acc: ParentChildrenTuple<R>[], row: GridDataRow<R>, parentMatches?: boolean): ParentChildrenTuple<R>[] {
```

Comment:
Thought: I'm wondering if it'd be slightly cleaner to keep `filterFn` as it was, and then add a new function above this like:

```
function acceptAll(acc: ..., row: ...): ... {
  acc.concat([row, row.children.reduce(acceptAll, []) ?? []);
}
```

And then maybe change the logic to something like:

```
if (matches) {
  return acc.concat([row, row.children?.reduce(acceptAll. []));
} else {
  const matchedChildren = row.children?.reduce(filterFn, []);
  if (matchedChildren.length) {
     return acc.concat([row, matchedChildren]);
  }
  return acc;
}
```

Dunno, maybe it's not a big difference, wdyt?